Tests/ToolBox/PsychSignalDataTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace QuantConnect.Tests.ToolBox
+{
+    [TestFixture]
+    public class PsychSignalDataTests
+    {
+        [Test, Ignore("This test requires raw PsychSignal data")]
+        public void FileHourMatchesDataTimeRealRawData()
+        {
+            var rawPath = Path.Combine(Globals.DataFolder, "alternative", "psychsignal", "raw-psychsignal");
+            
+            foreach (var file in Directory.GetFiles(rawPath, "*.csv", SearchOption.TopDirectoryOnly).ToList())
+            {
+                var fileSplit = file.Split('_');
+
+                var hour = Convert.ToInt32(fileSplit[fileSplit.Length - 4]);
+
+                // Read one line of the file, and compare the hour of the day to the hour on the file
+                var line = File.ReadLines(file).ToList().Last().Split(',');
```

Comment:
This `ToList()` is not required. If we're just interested in the last line then we can keep it as `IEnumerable` so we don't need to read the _entire_ file into memory. With this file type it may be inconsequential, but with larger files it can cause an issue.